DiamondLightSource/python-artemis
# 368 generalise ispyb handling to rotation scans

Fixes #368

### To test:
- Run the new system test in `test_rotation_scan_plan.py`
- verify that results appear in https://ispyb-test.diamond.ac.uk/dc/visit/cm31105-4
